core/src/androidMain/kotlin/gatt/Callback.kt

```diff
@@ -46,7 +46,7 @@ internal class Callback(
 ) : BluetoothGattCallback() {
 
     private var disconnectedAction: DisconnectedAction? = null
-    fun invokeOnDisconnected(action: () -> Unit) {
+    fun invokeOnDisconnected(action: DisconnectedAction) {
```

Comment:
Just curious, is DisconnectedAction just an alias for () -> Unit?

Reply:
Yup

https://github.com/JuulLabs/kable/blob/226102b47e271a434ed865e72c72053e35a0c8c1/core/src/androidMain/kotlin/gatt/Callback.kt#L37